pkg/algorithm/nodeselect/v1alpha1/select_node.go

```diff
@@ -187,6 +210,96 @@ func (ac *Config) getBlockDevice() (*ndmapis.BlockDeviceList, error) {
 	if err != nil {
 		return nil, err
 	}
-	bdl := bdL.Filter(blockdevice.FilterClaimedDevices, blockdevice.FilterNonInactive)
+	bdl := bdL.Filter(blockdevice.FilterNonInactive)
```

Comment:
this filter need to be different for manual and auto cases

Reply:
No need